crates/bevy_render/src/view/mod.rs

```diff
@@ -1050,10 +1050,14 @@ pub fn prepare_view_targets(
     for (entity, camera, view, texture_usage, msaa) in cameras.iter() {
         let (Some(target_size), Some(target)) = (camera.physical_target_size, &camera.target)
         else {
+            commands.entity(entity).try_remove::<ViewTarget>();
```

Comment:
I'm not sure this one is necessary? In my testing when it happens it doesn't cause a crash.

Reply:
This is required because when this happens it skips the check to resolve the view_target_attachment.

So if the user closes the window and removes the camera's target in the same frame then it will cause the same crash this PR is fixing.